features/apimgt/org.wso2.carbon.apimgt.gateway.feature/pom.xml

```diff
@@ -193,11 +197,13 @@
                                 <bundleDef>net.sf.ehcache.wso2:ehcache</bundleDef>
                                 <bundleDef>org.json.wso2:json:${orbit.version.json}</bundleDef>
                                 <bundleDef>org.apache.commons:commons-pool2:${apache.commons.version}</bundleDef>
+                                <bundleDef>org.wso2.orbit.re2j:re2j</bundleDef>
```

Comment:
Think we need to define the version as well

Reply:
Resolved